src/main/java/seedu/duke/project/Project.java

```diff
@@ -0,0 +1,59 @@
+package seedu.duke.project;
+
+import java.time.LocalDate;
+
+public class Project {
+
+    public static ProjectBacklog backlog;
+    public static ProjectMember member;
```

Comment:
"member" can be changed to "members" which will make more sense as it is a list of members.

Reply:
Yes i forgot to incorporate that change will do